frontend/packages/kubevirt-plugin/integration-tests/tests/vm.detail.boot-order.scenario.ts

```diff
@@ -0,0 +1,133 @@
+import { browser } from 'protractor';
+import * as _ from 'lodash';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { click, waitForCount } from '@console/shared/src/test-utils/utils';
+import { isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import * as bootOrderView from '../views/editBootOrderView';
+import { createResource, deleteResource } from '@console/shared/src/test-utils/utils';
+import { getBootableDevices, getBootableDevicesInOrder } from '../../src/selectors/vm/devices';
+import { VM_CREATE_AND_EDIT_TIMEOUT_SECS } from './utils/consts';
+import { VirtualMachine } from './models/virtualMachine';
+import { getVMManifest } from './utils/mocks';
+import { getRandStr, getResourceObject, getSelectOptions, selectOptionByText } from './utils/utils';
+
+describe('KubeVirt VM detail - Boot Order Dialog', () => {
+  const JS_HTML5_DND =
+    'function e(e,t,n,i){var r=a.createEvent("DragEvent");r.initMouseEvent(t,!0,!0,o,0,0,0,c,g,!1,!1,!1,!1,0,null),Object.defineProperty(r,"dataTransfer",{get:function(){return d}}),e.dispatchEvent(r),o.setTimeout(i,n)}var t=arguments[0],n=arguments[1],i=arguments[2]||0,r=arguments[3]||0;if(!t.draggable)throw new Error("Source element is not draggable.");var a=t.ownerDocument,o=a.defaultView,l=t.getBoundingClientRect(),u=n?n.getBoundingClientRect():l,c=l.left+(l.width>>1),g=l.top+(l.height>>1),s=u.left+(u.width>>1)+i,f=u.top+(u.height>>1)+r,d=Object.create(Object.prototype,{_items:{value:{}},effectAllowed:{value:"all",writable:!0},dropEffect:{value:"move",writable:!0},files:{get:function(){return this._items.Files}},types:{get:function(){return Object.keys(this._items)}},setData:{value:function(e,t){this._items[e]=t}},getData:{value:function(e){return this._items[e]}},clearData:{value:function(e){delete this._items[e]}},setDragImage:{value:function(e){}}});if(n=a.elementFromPoint(s,f),!n)throw new Error("The target element is not interactable and need to be scrolled into the view.");u=n.getBoundingClientRect(),e(t,"dragstart",101,function(){var i=n.getBoundingClientRect();c=i.left+s-u.left,g=i.top+f-u.top,e(n,"dragenter",1,function(){e(n,"dragover",101,function(){n=a.elementFromPoint(c,g),e(n,"drop",1,function(){e(t,"dragend",1,callback)})})})})';
```

Comment:
@yaacov @suomiy  I'm reeeeally not sure about this :D I'm not able to get the DnD working with protractor, apparently it doesn't support HTML5 Drag and drop....
I took this from https://github.com/html-dnd/html-dnd
Opinions?

Reply:
can't we rather use html-dnd and add it as a dependency?

Reply:
Yeah, how can I add it as a dependency?

Reply:
```bash
cd frontend
yarn add --dev html-dnd -W
```